templates/DO-178C/README.md

```diff
@@ -1,3 +1,35 @@
 # StrictDoc template for DO-178C standard
 
-TBD: Instructions how to use.
+This DO-178C template provides an overview of the documents required as acceptable mean of compliance.
+
+The aim of the project is also to demonstrate the use of strictdoc integrating:
+- diagrams (asciiflow)
```

Comment:
I checked the name of the tools and it seems to be `ASCIIFlow`. Please rename `asciiflow` -> `ASCIIFlow`.